dash/backend/src/util/tablify.ts

```diff
@@ -0,0 +1,43 @@
+export function tablify(obj: any, depth = 0, maxdepth = 10) {
```

Comment:
Where is this function from?
Wondering if we should c+p the Tablify class from the frontend. I did a bit of bug fixing on it :)

Reply:
It looks a bit similar to this - https://github.com/m9sweeper/m9sweeper/blob/974365e590265f452b62c1b00facf1b94820d79a/dash/frontend/src/app/core/pipe/tablify.pipe.ts#L6

Reply:
Though I don't think we want the backend to import PipeTransform, so we wouldn't copy paste the whole thing - just parts of it (maybe that's what Charis did). @becky-intelletive  Do you see a specific change required here?

Reply:
You're right. On closer inspection it looks like it was a c+p :)

Reply:
@jacobbeasley The frontend pipe was from Angular. Per the instruction from the relevant ticket, this was modified a bit to provide the same usage. Let me know if you want any further changes.